index.js

```diff
@@ -16,7 +16,9 @@ function exportToJsonString(idbDatabase, cb) {
     );
     transaction.onerror = (event) => cb(event, null);
 
-    Array.from(idbDatabase.objectStoreNames).forEach((storeName) => {
+    const objectStoreNames = Array.from(new Set(idbDatabase.objectStoreNames));
```

Comment:
Please could you explain how there can be duplicates?

Reply:
I think it's better to remove this part because  I couldn't reproduce the error